packages/wrangler/src/cloudchamber/client/core/README.md

```diff
@@ -0,0 +1,8 @@
+# What is this?
```

Comment:
we shouldnt include this as it doesnt apply anymore, mb as the codegeneration code should've been updated to not include this!